### PR TITLE
Support release tag in openjdk regression test

### PR DESCRIPTION
**Pull request by llxia**
Issue: #678

Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
Tested in Grinder:
- build without `RELEASE_TAG`:
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/16/
- Tested `jdk_util_0` with `RELEASE_TAG` set to `jdk-11.0.1+13`: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/19/
Note: `CurrencyTest` and `ValidateISO4217` failed with material that is tagged with `jdk-11.0.1+13`

File: openjdk_regression/build.xml [hunk 1]
```diff
@@ -36,6 +36,17 @@
         <equals arg1="${JDK_VERSION}" arg2="11"/>
     </condition>
 	
+	<property environment="env" />
```
Comment (karianna) [inline openjdk_regression/build.xml]: Is the whitespace out a bit?

**Comment by llxia** [comment]
Output for git cmd with `RELEASE_TAH`:
```
00:03:10.858 getOpenjdk:
00:03:10.858      [echo] Run git cmd: clone --depth 1 -q --branch jdk-11.0.1+13 https://github.com/AdoptOpenJDK/openjdk-jdk11u.git
00:04:33.369      [exec] Note: checking out '45815ba40106bff994ca5b70030571b9f980cef4'.
...
```

**Comment by karianna** [comment]
Thank you!

**Pull request merged**